admin/tabs/wizard.php

```diff
@@ -67,41 +57,26 @@ public function main_column() {
                   //{
                     //$key++;
         ?>
-
-        <table class="widefat striped">
-          <thead>
-          <tr>
-            <th><?php echo esc_html( $config->steps[$step -1]->name )?></th>
-          </tr>
-          </thead>
-          <tbody>
-          <tr>
-            <td>
-              <?php echo wp_kses_post( $parsedown->text( $config->steps[$step -1]->description ) )?>
-            </td>
-          </tr>
-          </tbody>
-        </table>
-        <br>
-
+        <?php echo esc_html( $config->steps[$step -1]->name )?>
```

Comment:
Can we wrap the step name in an `<h1>`? It will look more like a heading/title then.